apps/crn-server/test/app.test.ts

```diff
@@ -0,0 +1,79 @@
+import supertest from 'supertest';
+import { authHandlerMock } from './mocks/auth-handler.mock';
+
+import { NewsDataProvider } from '../src/data-providers/types';
+
+describe('app is set with correct data provider for news', () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    jest.resetAllMocks();
+    process.env = { ...OLD_ENV };
```

Comment:
Without the resetAllMocks, the `newsSquidexDataProviderMock.fetch` seems that is called more than once in the tests that came after the first one that has a call to `newsSquidexDataProviderMock.fetch`.

And without `resetModules` even in the test where the calls to `newsSquidexDataProviderMock.fetch` should be 0, it appears as not zero because of the cache of previous tests.